crates/wit-smith/src/lib.rs

```diff
@@ -24,7 +24,14 @@ pub fn smith(config: &Config, u: &mut Unstructured<'_>) -> Result<Vec<u8>> {
         let unresolved = pkg.sources.parse().unwrap();
         let id = match resolve.push(unresolved) {
             Ok(id) => id,
-            Err(e) => panic!("bad wit parse: {e:?}"),
+            Err(e) => {
+                if e.to_string()
+                    .contains("both an imported and exported copy of the same interface")
```

Comment:
This error message was [changed](https://github.com/bytecodealliance/wasm-tools/pull/1081/commits/1f136b68880da181d5e2f626d592b684d5264a7b) to "an interface in incompatible ways".